main.js

```diff
@@ -11,23 +11,24 @@ var gridBoard = document.querySelector(".game-board")
 gridBoard.addEventListener('click', function(event) {
     if (event.target.matches('.grid-item')) {
       turnGridArray()
-      tokenText()
-      checkGridItem()
+      toeknSwitch()
```

Comment:
This function is spelled wrong compared to line 23! Make sure this is corrected to make sure game still functions properly.